.github/workflows/main.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   build:
     strategy:
       matrix:
-        configuration: [Debug, Release]
+        configuration: [Debug, Release]    
     env:
       Solution_Name: src\Notepads.sln
       Project_Path: src\Notepads\Notepads.csproj
```

Comment:
remove Project_Path variable, since it is not used anywhere

Reply:
definitely not